partshop-backend/src/main/java/hu/kosztaauto/partshop/dto/AlkatreszWithWarehouseDTO.java

```diff
@@ -0,0 +1,18 @@
+package hu.kosztaauto.partshop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AlkatreszWithWarehouseDTO {
+    private String cikkszam;
+    private String megnevezes;
+    private String autoTipus;
+    private int ar;
+    private RaktarDTO raktar;
```

Comment:
Wouldnt refering only to the warehouses id be better?

Reply:
I prefer DTO's because they provide reusable schemes and can customise them according to what I need to return, so you don't have to filter it on frontend